### PR TITLE
docs: add terms of service

### PR DESCRIPTION
**Pull request by heyjay44**
- [x] add two links (line 2 and line 26)
- [x] update date (line 2) 
- [x] check formatting
- [x] check links
- [x] need to move .md file to correct part of the directory tree - >src/pages/docs/terms.md
- [x] Figure out plans links

### CODE REVIEW

File: src/terms [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# **[Terms of service](https://web3.storage/terms/#terms-of-service)**
```
Comment (olizilla) [inline src/terms:1]: let headings be headings
```suggestion
# Terms of service
```

File: src/terms [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+[If you are still using the legacy web3.storage API (e.g., you created an account before November xx, 2023), please view the applicable Terms of Service here(ADD LINK).]
```
Comment (olizilla) [inline src/terms:2]: replace `xx` with a date

Comment (olizilla) [reply src/terms:2]: and ADD LINK

File: src/terms [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+The following terms and conditions govern all use of the web3.storage website (“web3.storage” or the “Website”), the w3link/w3s.link gateway (“Gateway”) and all content, services and products available at or through the Website or Gateway (collectively, “Services”). The Services are offered subject to your acceptance without modification of all of the terms and conditions contained herein. As all data uploaded to web3.storage will be available via IPFS, the Services are also subject to the [Terms of Service of IPFS.io](https://discuss.ipfs.tech/tos) and [Privacy Policy of IPFS.io](https://discuss.ipfs.tech/privacy).
+
+If you do not agree to all the terms and conditions of this agreement (“Agreement” or “TOS”), then you may not access the Website or use any of the Services.
+
+Nitro Data LLC (“we,” “us”) may make changes to this Agreement and pricing from time to time. If you do not agree to these revisions, you may stop using the Services.
+
+#### **[Storage term](https://web3.storage/terms/#storage-term)**
```
Comment (olizilla) [inline src/terms:10]:
```suggestion
## Storage term
```

File: src/terms [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+We reserve the right to terminate the Services at our sole discretion or to transfer operation of the Services to a third party or smart contract. Prior to termination or transfer, we will provide 90 days’ notice to users via email to allow users enough time to make arrangements if any (e.g. remote pinning to their own IPFS node, remote pinning to another service, or preparing to manage the renewal of their Filecoin deals on their own).
+
+All data uploaded to web3.storage is available to anyone who requests it using the correct CID. **Users should not store any private or sensitive information in an unencrypted form using web3.storage.** Deleting files from web3.storage via the site's web console or API will remove them from the file listing for a user's account, but you may be charged for at least 30 days of storage for those files. Additionally, uploaded files typically remain on our infrastructure for at minimum thirty (30) days, and nodes on the IPFS network may retain copies of the data indefinitely. **Users should not use web3.storage to store data that may need to be permanently deleted in the future.**
+
+#### **Paid Accounts**
```
Comment (olizilla) [inline src/terms:19]:
```suggestion
## Paid Accounts
```

File: src/terms [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+<u>Billing.  </u>
```
Comment (olizilla) [inline src/terms:21]: why underscore billing? can use an h3?

File: src/terms [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+ In order to begin storing files via the Services, you will need to provide us with one or more Payment Methods. "Payment Method" means a current, valid method of payment, which may involve a third party or payment processing vendor. You can increase your storage limit and add paid features to your account by signing up for a paid subscription plan (“Paid Plan”). 
+
+You agree that you’ll pay the amounts set forth in https://web3.storage/account/payment/ for your chosen tier (“Subscription Charge”) plus any charges for additional usage beyond the storage limit of your selected tier during the previous month (“Overages”) (collectively, “Subscription Charge” and “Overages” are referred to as (the “Fees”) for each billing cycle). 
+
+When determining tier storage limits and Overages, we track usage for payment using the volume of data associated with shard content identifiers/CIDs (CIDs referencing the actual data sent to web3.storage and stored) – usage is not tracked based on content CIDs (CIDs used to reference and access uploads in a format useful to users, like specific files and directories). See more detail on this point here(ADD LINK).
```
Comment (olizilla) [inline src/terms:26]: ADD LINK

File: src/terms [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+You authorize us to charge any Payment Method associated with your account in case your primary Payment Method is declined or no longer available to us for payment of Fees you owe us. You remain responsible for any uncollected Fees. If a payment is not successfully settled, due to expiration, insufficient funds, or otherwise, and you do not cancel your account, we may suspend your access to the Paid Plan until we have successfully charged a valid Payment Method. For some Payment Methods, the issuer may charge you additional fees, such as foreign transaction fees or other fees relating to the processing of your Payment Method.
+
+<u>Cancellation.  </u>
```
Comment (olizilla) [inline src/terms:30]: these all look like h3 items
```suggestion
### Cancellation.
```

File: src/terms [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+
+<u>Taxes.  </u>
+The Fees do not include any taxes, levies, duties, or similar governmental assessments, including value-added, sales, use, or withholding taxes assessable by any domestic or foreign jurisdiction (“Taxes”). You agree that you’re responsible for any applicable Taxes, and that you will provide us with any applicable tax identification information that we may require by law to ensure our compliance with applicable tax regulations and authorities in applicable jurisdictions.
+
+<u>Overages. </u>
+You agree to pay the following Overages fees for storage amounts exceeding your selected tier (for more information on tiers, see https://web3.storage/account/payment/). Overages will be billed to you at the beginning of your next billing cycle.
+
+#### **Acceptable Use**
```
Comment (olizilla) [inline src/terms:45]:
```suggestion
## Acceptable Use
```

**Review by olizilla — CHANGES_REQUESTED** [review]
this should live at `src/pages/docs/terms.md` (needs .md extention)
headers need fixing. links and dates filling out.

**Comment by olizilla** [comment]
@heyjay44 are you using a tool to make the markdown? it's got the same formatting errors as the docs content?

File: src/terms (hunk 1 above)
Comment (heyjay44) [reply src/terms:1]: I agree but I'm not sure if this is link is used elsewhere to reference/direct to this section.

File: src/terms (hunk 3 above)
Comment (heyjay44) [reply src/terms:10]: Same as above.

File: src/terms (hunk 5 above)
Comment (heyjay44) [reply src/terms:21]: Just copying the formatting from the document that was approved.

File: src/terms (hunk 7 above)
Comment (heyjay44) [reply src/terms:30]: Again, copying format from received document. It was underlined.

File: src/terms (hunk 6 above)
Comment (heyjay44) [reply src/terms:26]: @dchoi27 which reference link did you have in mind?

**Comment by heyjay44** [comment]
@olizilla I'm just using vscode and the formatting from the previous ToS.

File: src/terms (hunk 6 above)
Comment (dchoi27) [reply src/terms:26]: ah good catch. it should be a link to a section of docs explaining store vs upload. we can remove for launch if too much to coordinate and put back after

Comment (vasco-santos) [reply src/terms:26]: I am removing for now until we have docs for this, maybe a FAQ

File: src/pages/docs/terms.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+
+### Cancellation
+
+You can cancel your Paid Plan at any time, and you will be able to upload new Data to web3.storage through the end of your billing period. Payments are nonrefundable and there are no refunds or credits for partially used membership periods. After cancellation, data uploaded via a Paid Plan may not be available through our hosted IPFS infrastructure after the end of your billing period.
+
+### Plan Adjustments
+
+You may change your chosen tier at any point. You will be charged for the tier you’re subscribed to at the time of billing (plus any Overages for stored content over that tier’s limit). By selecting your tier, you consent to the limits associated with your chosen tier (as set forth in https://web3.storage/account/payment/); you also consent to the Overages described in this Agreement.
```
Comment (vasco-santos) [inline src/pages/docs/terms.md:39]:
```suggestion
You may change your chosen tier at any point. You will be charged for the tier you’re subscribed to at the time of billing (plus any Overages for stored content over that tier’s limit). By selecting your tier, you consent to the limits associated with your chosen tier (as set forth in https://web3.storage/pricing/); you also consent to the Overages described in this Agreement.
```

File: src/pages/docs/terms.md [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+
+We reserve the right to change our Paid Plans or adjust pricing for our service or any components thereof in any manner and at any time, in our sole discretion and without notice. If we adjust the pricing of your Paid Plan, we will provide you with at least fourteen (14) days’ notice before these changes take effect. By continuing to use the Services, you agree to terms of this Agreement (including any updated terms).
+
+### Taxes
+
+The Fees do not include any taxes, levies, duties, or similar governmental assessments, including value-added, sales, use, or withholding taxes assessable by any domestic or foreign jurisdiction (“Taxes”). You agree that you’re responsible for any applicable Taxes, and that you will provide us with any applicable tax identification information that we may require by law to ensure our compliance with applicable tax regulations and authorities in applicable jurisdictions.
+
+### Overages
+
+You agree to pay the following Overages fees for storage amounts exceeding your selected tier (for more information on tiers, see https://web3.storage/account/payment/). Overages will be billed to you at the beginning of your next billing cycle.
```
Comment (vasco-santos) [inline src/pages/docs/terms.md:51]:
```suggestion
You agree to pay the following Overages fees for storage amounts exceeding your selected tier (for more information on tiers, see https://web3.storage/pricing/). Overages will be billed to you at the beginning of your next billing cycle.
```

File: src/pages/docs/terms.md [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+
+## Paid Accounts
+
+### Billing
+
+In order to begin storing files via the Services, you will need to provide us with one or more Payment Methods. "Payment Method" means a current, valid method of payment, which may involve a third party or payment processing vendor. You can increase your storage limit and add paid features to your account by signing up for a paid subscription plan (“Paid Plan”).
+
+You agree that you’ll pay the amounts set forth in https://web3.storage/account/payment/ for your chosen tier (“Subscription Charge”) plus any charges for additional usage beyond the storage limit of your selected tier during the previous month (“Overages”) (collectively, “Subscription Charge” and “Overages” are referred to as (the “Fees”) for each billing cycle).
```
Comment (vasco-santos) [inline src/pages/docs/terms.md:27]:
```suggestion
You agree that you’ll pay the amounts set forth in https://web3.storage/pricing/ for your chosen tier (“Subscription Charge”) plus any charges for additional usage beyond the storage limit of your selected tier during the previous month (“Overages”) (collectively, “Subscription Charge” and “Overages” are referred to as (the “Fees”) for each billing cycle).
```

**Review by vasco-santos — APPROVED** [review]

**Pull request merged**